src/components/App/index.js

```diff
@@ -1,68 +1,45 @@
-import React, { useState } from 'react';
-import './App.css';
+import React from 'react';
+import './index.css';
 
+import Container from 'react-bootstrap/Container'
+import Form from '../Form';
+import ListWrapper from '../ListWrapper';
 import config from '../../config.json';
-import Button from 'react-bootstrap/Button';
-import Container from 'react-bootstrap/Container';
-import List from '../List';
-import Toggle from '../Toggle';
 
-function App() {
-  // TODO: update responseEmail in config
+export default function App () {
   const {
-    formLink,
-    responseEmail,
     categories,
+    categoryDisplayTitles,
+    filters,
+    filterDisplayTitles,
     offerString,
-    requestString
+    requestString,
+    responseEmail
   } = config;
-  const toggleOptions = ['all', 'needs', 'offers'];
-  const [type, setType] = useState('all');
+  const path = window.location.pathname;
 
   return (
     <Container className="App">
-      <header className="border-bottom mb-3">
-        <h1>LMCC Community Assistance</h1>
-        <p className="lead">A place to offer and ask for help</p>
+      <header className="border-bottom mb-3 mt-3">
+        <h1>LMCC Needs/Provides Forum</h1>
+        <p className="lead">A place for the <a className="info" href="https://www.lowermanhattanchurch.com/">Lower Manhattan Community Church</a> community to share and meet our needs</p>
       </header>
-      <div className="border-bottom mb-3">
-        <Button
-          className="mr-2 mb-3"
-          href={formLink}
-          variant="outline-secondary"
-        >
-          Submit a need or offer
-        </Button>
-        <Button
-          className="mb-3"
-          href={`mailto:${responseEmail}`}
-          variant="outline-secondary"
-        >
-          Email us to respond to a listing
-        </Button>
-      </div>
-      <h2>Listings</h2>
-      <Toggle
-        handleChange={setType}
-        options={toggleOptions}
-        prependOptionText="See "
-        selected={type}
-      />
-      <List
-        categories={categories}
-        offerString={offerString}
-        requestString={requestString}
-        type={type}
-      />
-      <Button
-        className="mb-3"
-        href={`mailto:${responseEmail}`}
-        variant="outline-secondary"
-      >
-        Email us to respond to a listing
-      </Button>
+      {
+        path === '/submit' &&
+        <Form />
+      }
+      {
+        path === '/' &&
```

Comment:
You might want to handle `path` being anything else as well. Maybe use `history.replaceState()` to send them to `/`.

Reply:
Good idea.